util/taskcluster.py

```diff
@@ -34,6 +35,11 @@ def report_progress(count, block_size, total_size):
     assert target_dir is not None
 
     target_dir = os.path.abspath(target_dir)
+    try:
+        os.makedirs(target_dir)
+    except OSError as e:
+        if e.errno != errno.EEXIST:
```

Comment:
Can we ensure this is okay on all Python versions? I know some may have variances on the properties available on the exceptions.

Reply:
It is, I checked.